accounts/urls.py

```diff
@@ -14,9 +17,17 @@
 app_name = "accounts"
 
 
+def get_login_view():
+    return DevLoginView if settings.IS_DEV_LOGIN else LoginView
+
+
+def get_logout_view():
+    return DevLogoutView if settings.IS_DEV_LOGIN else LogoutView
```

Comment:
I think we could simplify this to something like:
`FinalLogoutView = DevLogoutView if settings.IS_DEV_LOGIN else LogoutView`